all-actors/src/main/java/org/sunbird/cert/actor/CertificateGeneratorActor.java

```diff
@@ -23,114 +23,117 @@
-			//properties.put(JsonKey.AZURE_STORAGE_KEY,System.getenv(JsonKey.AZURE_STORAGE_KEY));
+            //HashMap<String,String> properties = new HashMap<>();
+            //properties.put(JsonKey.CONTAINER_NAME,System.getenv(JsonKey.CONTAINER_NAME));
+            //properties.put(JsonKey.CLOUD_STORAGE_TYPE,System.getenv(JsonKey.CLOUD_STORAGE_TYPE));
+            //properties.put(JsonKey.CLOUD_UPLOAD_RETRY_COUNT,System.getenv(JsonKey.CLOUD_UPLOAD_RETRY_COUNT));
+            //properties.put(JsonKey.AZURE_STORAGE_SECRET,System.getenv(JsonKey.AZURE_STORAGE_SECRET));
+            //properties.put(JsonKey.AZURE_STORAGE_KEY,System.getenv(JsonKey.AZURE_STORAGE_KEY));
 
-			//StorageParams storageParams = new StorageParams(properties);
-			//storageParams.init();
-			//return storageParams.upload(System.getenv(JsonKey.CONTAINER_NAME), "/", file, false);
+            //StorageParams storageParams = new StorageParams(properties);
+            //storageParams.init();
+            //return storageParams.upload(System.getenv(JsonKey.CONTAINER_NAME), "/", file, false);
 
-			File file = FileUtils.getFile("conf/certificate/"+certFileName);
-			return AzureFileUtility.uploadFile(System.getenv(JsonKey.CONTAINER_NAME),file);
-		}catch (Exception ex) {
-			logger.info("CertificateGeneratorActor:upload: Exception occurred while uploading certificate.",ex);
-		}
-		return "";
-	}
+            File file = FileUtils.getFile("conf/certificate/" + certFileName);
+            return AzureFileUtility.uploadFile(System.getenv(JsonKey.CONTAINER_NAME), file);
+        } catch (Exception ex) {
+            logger.info("CertificateGeneratorActor:upload: Exception occurred while uploading certificate.", ex);
+        }
+        return "";
+    }
 
-	private HashMap<String,String> getProperties(){
-		// properties need to populate from env
-		HashMap<String,String> properties = new HashMap<>();
-		properties.put(JsonKey.DOMAIN_PATH,System.getenv(JsonKey.DOMAIN_PATH));
-		properties.put(JsonKey.ASSESSED_DOMAIN,System.getenv(JsonKey.ASSESSED_DOMAIN));
-		properties.put(JsonKey.BADGE_URL,System.getenv(JsonKey.BADGE_URL));
-		properties.put(JsonKey.ISSUER_URL,System.getenv(JsonKey.ISSUER_URL));
-		properties.put(JsonKey.TEMPLATE_URL,System.getenv(JsonKey.TEMPLATE_URL));
-		properties.put(JsonKey.CONTEXT,System.getenv(JsonKey.CONTEXT));
-		properties.put(JsonKey.VERIFICATION_TYPE,System.getenv(JsonKey.VERIFICATION_TYPE));
-		properties.put(JsonKey.ACCESS_CODE_LENGTH,System.getenv(JsonKey.ACCESS_CODE_LENGTH));
-		logger.info("CertificateGeneratorActor:getProperties:properties got from env ".concat(Collections.singleton(properties.toString())+""));
-		return properties;
-	}
+    private HashMap<String, String> getProperties() {
+        // properties need to populate from env
+        HashMap<String, String> properties = new HashMap<>();
+        properties.put(JsonKey.DOMAIN_PATH, System.getenv(JsonKey.DOMAIN_PATH));
```

Comment:
better keep this in try catch and log exactly at which env variable exception is thrown